internal/service/privatelinkendpointserverless/resource_privatelink_endpoint_serverless.go

```diff
@@ -79,11 +80,7 @@ func resourceCreate(ctx context.Context, d *schema.ResourceData, meta any) diag.
 	projectID := d.Get("project_id").(string)
 	instanceName := d.Get("instance_name").(string)
 
-	serverlessTenantCreateRequest := &admin.ServerlessTenantCreateRequest{
-		Comment: conversion.StringPtr("create"),
```

Comment:
there is not comment attribute in resource schema that can be passed in create?

Reply:
I believe there is not. I am curious if we have context on why this hardcoded value is defined.

Reply:
>I believe there is not. I am curious if we have context on why this hardcoded value is defined.

Yeah I couldn't figure it out, I didn't get any such details from the API team as well on this

Reply:
>Aastha if we decided to add this comment field in the schema, would the problem of drift still persist I guess?

Do you mean add it to `mongodbatlas_privatelink_endpoint_serverless` schema? The drift detection currently happens in the other resource which is `mongodbatlas_privatelink_endpoint_service_serverless` so the issue will persist.